platform-includes/crons/setup/javascript.nextjs.mdx

```diff
@@ -3,9 +3,14 @@ Cron monitoring is only supported in Server and Edge runtimes for Next.js
 ## Automatic Check-Ins (Vercel Only)
 
 If you are hosting your Next.js application on Vercel and you are using [Vercel's Cron Jobs feature](https://vercel.com/docs/cron-jobs), you can configure the Next.js SDK to automatically create Check-Ins for you.
-Instrumented cron jobs are decided at runtime by examining the `crons` field in your `vercel.json` file.
 
-Set the `automaticVercelMonitors` option to `true` in your Sentry settings in `next.config.js`. See <PlatformLink to="/manual-setup/#opt-out-of-auto-instrumentation-of-vercel-cron-jobs">Manual Setup</PlatformLink> for more details.
+Vercel will create cron jobs depending on the `crons` field in your `vercel.json`. Set the `automaticVercelMonitors` option to `true` in your Sentry settings in `next.config.js` to automatically create cron job monitoring for them.
+
+<Alert level="warning">
+Automatic instrumentation of Vercel cron jobs currently only works for the Pages Router. App Router route handlers are not yet supported.
+</Alert>
```

Comment:
Let's add this alert/warning but leave the rest as-is. People with Vercel crons will know what's going on here.